x-pack/plugins/uptime/server/lib/requests/get_journey_details.ts

```diff
@@ -38,6 +38,7 @@ export const getJourneyDetails: UMElasticsearchQueryFn<
       },
     },
     size: 1,
+    _source: ['synthetics', 'monitor', '@timestamp'],
```

Comment:
Basically, I was writing an optimization PR and realized we were only using these fields, so I figured it'd marginally reduce the size of the server responses. In hindsight it's probably not worth the effort, because the `_source` field is deprecated and it will create additional work for us in the future. I'll drop it.

Reply:
https://github.com/elastic/kibana/pull/103979/commits/257dd8303ac57f41426774d4a27f08e32cdf668c